### PR TITLE
Few SPDX info addition and REUSE lint to CI

### PR DESCRIPTION
**Pull request by ret2libc**
Current status:
```
# SUMMARY

* Bad licenses:
* Deprecated licenses:
* Licenses without file extension:
* Missing licenses:
* Unused licenses:
* Used licenses: Apache-2.0, BSD-2-Clause, BSD-3-Clause, CC-BY-SA-4.0, CC0-1.0, GPL-1.0-or-later, GPL-2.0-only, GPL-2.0-or-later, GPL-3.0-or-later, LGPL-2.0-or-later, LGPL-2.1-only, LGPL-2.1-or-later, LGPL-3.0-only, MIT, MPL-1.1, MS-PL, NCSA, RSA-MD
* Read errors: 0
* Files with copyright information: 2854 / 3076
* Files with license information: 2854 / 3076

Unfortunately, your project is not compliant with version 3.0 of the REUSE Specification :-(
```

Partially addressing https://github.com/rizinorg/rizin/issues/683

### CODE REVIEW

File: librz/bin/format/spc700/spc_specs.h [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// SPDX-FileCopyrightText: maijin <maijin@malware.lu>
```
Comment (XVilka) [inline librz/bin/format/spc700/spc_specs.h]: @Maijin do you want to use another email here?

File: librz/core/cmd_linux_heap_glibc.c [hunk 2]
```diff
@@ -1,3 +1,6 @@
+// SPDX-FileCopyrightText: Pulak Malhotra
```
Comment (XVilka) [inline librz/core/cmd_linux_heap_glibc.c]: @PulakIIIT do you want to put your email here?

**Review by XVilka — APPROVED** [review]

File: librz/core/cmd_linux_heap_glibc.c (hunk 2 above)
Comment (MalhotraPulak) [reply librz/core/cmd_linux_heap_glibc.c]:
```suggestion
// SPDX-FileCopyrightText: Pulak Malhotra <pulakmalhotra2000@gmail.com>
```

File: librz/bin/format/spc700/spc_specs.h (hunk 1 above)
Comment (Maijin) [reply librz/bin/format/spc700/spc_specs.h]: Do I need to use an email?

Comment (Maijin) [reply librz/bin/format/spc700/spc_specs.h]: If not just remove it

Comment (ret2libc) [inline librz/bin/format/spc700/spc_specs.h]:
```suggestion
// SPDX-FileCopyrightText: maijin

**Pull request merged**